charts/incubator/speedtest-tracker/Chart.yaml

```diff
@@ -3,11 +3,7 @@ appVersion: "latest"
 dependencies:
   - name: common
     repository: https://library-charts.truecharts.org
-    version: 11.1.2
-  - condition: postgresql.enabled
-    name: postgresql
-    repository: https://deps.truecharts.org/
-    version: 11.0.31
+    version: 12.4.21
```

Comment:
lets please bump this also to the latest common

Reply:
bumped